apps/web/app/api/user/tokens/route.ts

```diff
@@ -31,7 +38,7 @@ export const GET = withSession(async ({ session }) => {
 
 // POST /api/user/tokens – create a new token for a specific user
 export const POST = withSession(async ({ req, session }) => {
-  const { name } = await req.json();
+  const { name } = await createApikeySchema.parseAsync(req.json());
```

Comment:
- Create API token doesn't work.
- I'm not sure why we have to use `parseAsync` instead of just `parse`.
- With this validation, you have to update the UI to parse the error response and display the error message.

Reply:
Done.